gateway-messages/src/sp_to_mgs.rs

```diff
@@ -210,6 +260,20 @@ pub struct RotImageDetails {
     pub version: ImageVersion,
 }
 
+/// This class exists for faux-mgs to nicely display the Firmware ID (FWID).
+#[derive(Clone, Debug)]
+#[must_use = "this struct does nothing unless displayed"]
+pub struct RotImageDetailsDisplay<'a>(pub &'a RotImageDetails);
+
+impl<'a> fmt::Display for RotImageDetailsDisplay<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let s = self.0;
+        write!(f, "digest: {}, ", &Digest256Display(&s.digest))?;
```

Comment:
(Same note about adding an `s.digest.display()` helper; I'll stop leaving this comment, but I think we should add them for all these display adapters.)

Reply:
I think I have dealt with all of these.